support-payment-api/src/main/scala/backend/PaypalBackend.scala

```diff
@@ -142,7 +142,7 @@ class PaypalBackend(
   }
 
 
-  private def getOrCreateIdentityIdFromEmail(email: String): Future[Option[IdentityIdWithGuestAccountCreationToken]] =
+  private def getOrCreateIdentityIdFromEmail(email: String): Future[Option[Long]] =
```

Comment:
identity id is surely an (opaque) String not a Long?

Reply:
you're right, though payment-api has always decoded to Long! I'll leave this for a separate PR